### PR TITLE
add multi-line (>250 chars) support for long boosts into IRC messages

### PR DESCRIPTION
**Pull request by dowodenum**
Pretty simple, chunks() separates a long string into a list of strings, each element having a length no longer than 250. Then iterate through that list and send each chunk as an IRC message.

### CODE REVIEW

File: src/irc/__init__.py [hunk 1]
```diff
@@ -183,3 +188,8 @@ def _new_message(data, value, numerology_func=number_to_numerology):
         data += " " + app
 
     return data
+
+
+def chunks(l, n):
```
Comment (valcanobacon) [inline src/irc/__init__.py]: Can you add typing to this, I think the following should work. You need to `from typing import List` too.

```suggestion
def chunks(message: str, n: int) -> List[str]:
```

File: src/irc/__init__.py [hunk 2]
```diff
@@ -183,3 +188,8 @@ def _new_message(data, value, numerology_func=number_to_numerology):
...
+    n = max(1, n)
+    return (l[i : i + n] for i in range(0, len(l), n))
```
Comment (valcanobacon) [inline src/irc/__init__.py]: This is longer but might be easier to read

```suggestion
    for i in range(0, len(message), n):
        yield message[i: i + n]
```

**Review by valcanobacon — CHANGES_REQUESTED** [review]
Thanks for this. Could you add some type annotations. A unit test for chunk would be great. Have you manually tested this?

**Comment by dowodenum** [comment]
image

**Review by valcanobacon — APPROVED** [review]
This is great thanks!

**Pull request merged**